src/Dfe.PlanTech.Web/Helpers/ComponentViewsFactory.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Dfe.PlanTech.Web.Helpers;
+
+public static class ComponentViewsFactory
+{
+    public static string GetViewForType(object model) => $"Components/{model.GetType().Name}";
```

Comment:
Not sure how well this will work out long term (e.g. what if we want different folders?), but for now this works with all existing components. Probably look at in future as/when components expand.